hack/prepare-e2e.sh

```diff
@@ -0,0 +1,108 @@
+
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OS=$(go env GOOS)
+ARCH=$(go env GOHOSTARCH)
+
+INSTALL_DIR=/usr/local/bin
+IMAGE_FILE=test/e2e/e2e.yaml
+
+prepare_ok=true
+# install e2e
+function install_e2e()
+{
+    if ! command -v e2e &> /dev/null; then
+      echo "please refer https://github.com/apache/skywalking-infra-e2e/blob/main/docs/en/contribution/Compiling-Guidance.md to install e2e!"
+      $prepare_ok=false
+    fi
+}
```

Comment:
This is no needed if https://github.com/apache/skywalking-swck/pull/46/files#r745608218